src/main/java/com/github/javafaker/service/RandomService.java

```diff
@@ -4,8 +4,13 @@
 
 public class RandomService {
     private static final Random SHARED_RANDOM = new Random();
+    private static final char[] hexValues;
     private final Random random;
 
+    static {
+        hexValues = new char[]{'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'};
```

Comment:
could you just inline this? I don't see the benefit of using a static block here